src/core/wagmi/createWagmiClient.ts

```diff
@@ -1,105 +1,127 @@
-import { createAsyncStoragePersister } from '@tanstack/query-async-storage-persister';
-import {
-  Chain,
-  CreateClientConfig,
-  configureChains,
-  createClient,
-  createStorage,
-} from 'wagmi';
-import { jsonRpcProvider } from 'wagmi/providers/jsonRpc';
+// import { Chain, Transport } from 'viem';
+// import { createConfig, http } from 'wagmi';
 
-import { proxyRpcEndpoint } from '../providers';
-import { queryClient } from '../react-query';
-import { SUPPORTED_CHAINS, getDefaultRPC } from '../references';
-import { LocalStorage } from '../storage';
-import { ChainId, chainHardhat, chainHardhatOptimism } from '../types/chains';
-import { findRainbowChainForChainId } from '../utils/chains';
+// // import { proxyRpcEndpoint } from '../providers';
+// // import { queryClient } from '../react-query';
+// import { SUPPORTED_CHAINS, getDefaultRPC } from '../references';
+// import { ChainId, chainHardhat, chainHardhatOptimism } from '../types/chains';
+// import { findRainbowChainForChainId } from '../utils/chains';
```

Comment:
Same here. Can we delete this or keep it ?

Reply:
removed the code in your comments 👌  thanks